guides/telemetry.md

```diff
@@ -62,7 +62,7 @@ This event is raised only on the master node.
 
 ```erlang
 event_name: [amoc, throttle, rate]
-measurements: #{rate := non_neg_integer()}
+measurements: #{rate_per_minute := float()}
```

Comment:
recently we had to stick to rate_per_minute because of metrics reporting, but since we moved exometer support from amoc amoc_arsenal, do we still want to stick to "per minute" value?